### ISSUE TITLE
Support for multiple space with different sync methods or easier account switching

### ISSUE THREAD

**Issue by Specki-Sh** [anyproto/anytype-ts#953]
### Have you read a contributing guide?

- [X] I have read CONTRIBUTING.md
- [X] I have searched the existing requests and didn't find any that were similar
- [X] I have considered creating a pull request instead and want to proceed

### Clear and concise description of the problem

I am using the app and encountered an issue when trying to set up two space. I want one space to be local and the other self-hosted. However, the app currently forces a complete logout and requires changing the sync type each time. 

### Suggested solution

Implement the ability to set different synchronization methods for specific space.

### Alternative

Provide a simpler way to switch between accounts without needing to fully log out or make other complex changes.

**Comment by xnm3YQNfEx** [comment]
Yes! I really would like this as well. I was looking to have some spaces that sync to the anytype network to sync to my phone, have some spaces that are self hosted, some that are local only.

If having the spaces with mixed networks isn't possible, then some change to enable easy switching between vaults on different networks would be a good alternative

**Comment by arran4** [comment]
Yes this would be awesome.

**Comment by asztrikx** [comment]
I would also love this, so that I could connect to others' self-hosted network.
